.travis.yml

```diff
@@ -44,14 +44,23 @@ script:
 
   elif [ "$TEST_TYPE" = build_website ]
   then
-
+    DOCS_VERSION=${TRAVIS_BRANCH%-stable}
```

Comment:
yeah, you are right, I'll move this whole logic with the deployment to Node so that we don't jump between Node and Bash all the time.
I have some positive experience using `https://www.npmjs.com/package/git-promise` for git checkouts and pushes.